__tests__/auth.test.ts

```diff
@@ -138,7 +158,50 @@ describe('auth tests', () => {
           <password>\${env.&amp;&lt;&gt;&quot;&apos;&apos;&quot;&gt;&lt;&amp;}</password>
         </server>
       </servers>
+      <profiles>
+        <profile>
+          <activation>
+            <activeByDefault>true</activeByDefault>
+          </activation>
+          <properties>
+            <gpg.passphrase>\${env.${gpgPassphrase}}</gpg.passphrase>
```

Comment:
putting this in a server section is more in line with the recommended setup at http://maven.apache.org/plugins/maven-gpg-plugin/usage.html#Configure_passphrase_in_settings.xml and also later on would allow for storing it in an encrypted fashion. Having it in a default profile as property is just too much overhead IMHO.